masschallenge/impact-api
# AC-4835: API improvements for BlackIron v1

[Ticket AC-4835](https://masschallenge.atlassian.net/browse/AC-4835)

Work done:
- Expanded /v1/users
- Expanded /v1/organizations
- Added program and cycle information to user history and organization history calls
- Added admin and primary contact information to user organizations and organization users calls
- Added /v1/industry
- Added industry data to /v1/users and /v1/organizations

Note that there is an explicit focus on just the GET and OPTIONS versions of the calls.  POST and PATCH have been deferred to [AC-4875](https://masschallenge.atlassian.net/browse/AC-4875).

To test:
- Review the output of the /v1/users, /v1/organizations, /v1/user/<id>/history, /v1/organization/<id>/history, /v1/user/<id>/organizations, /v1/organization/<id>/users, /v1/industry GET calls either through the browsable API or through Postman.
- Check the OPTIONS for these GET calls and confirm that they pretty much make sense.  Note that the OPTIONS for the list calls return a description of members of the list not a description of the list.  This will be addressed in [AC-4855](https://masschallenge.atlassian.net/browse/AC-4855).
